x-pack/plugins/fleet/server/services/epm/registry/requests.ts

```diff
@@ -88,15 +88,19 @@ function isSystemError(error: FailedAttemptErrors): boolean {
 }
 
 export function getFetchOptions(targetUrl: string): RequestInit | undefined {
+  const options: { [key: string]: any } = {
```

Comment:
Could we use `RequestInit` as a type here it's probably better than some any 


```suggestion
  const options: RequestInit = {
```